client/shipping-label/views/purchase/steps/rates/list.js

```diff
@@ -63,7 +61,7 @@ const ShippingRates = ( {
 
 	return (
 		<div>
-			{ renderRateNotice( showRateNotice ) }
+			{ shouldShowRateNotice && renderRateNotice() }
```

Comment:
Are you sure this is equivalent to the old code? How does the value `false` get rendered in this div?